staging/src/k8s.io/component-base/metrics/testutil/metrics_test.go

```diff
@@ -533,6 +533,7 @@ func TestGetHistogramVecFromGatherer(t *testing.T) {
 						{CumulativeCount: uint64Ptr(1), UpperBound: pointer.Float64Ptr(2.0)},
 						{CumulativeCount: uint64Ptr(1), UpperBound: pointer.Float64Ptr(5.0)},
 					},
+					CreatedTimestamp: timestamp.New(time.Unix(1705044926, 206877000)),
```

Comment:
This seems to cause CI failures as `timestamp` and `time` are not imported.

Reply:
Thank you, it has been changed